cmd/snippets.go

```diff
@@ -30,6 +32,9 @@ var (
 		Long:  "Download contents of the specified SQL snippet.",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if !utils.UUIDPattern.MatchString(args[0]) {
+				return errors.Errorf("snippet ID %q is not a UUID", args[0])
+			}
```

Comment:
```suggestion
```

We can just rely on the uuid.Parse called inside Run

Reply:
Yeah, added it here so it's 1-1 with the other places where we validate UUID format like here: https://github.com/supabase/cli/blob/feat/add-track-activity-query-size-setting/cmd/sso.go#L63